git_pile/git_pile.py

```diff
@@ -1874,7 +1874,7 @@ def cmd_reset(args):
         print("\nPile synchronized with current remote.")
 
 
-def parse_args(cmd_args):
+def parse_args(cmd_args, config):
```

Comment:
did you mean "config object" in the commit message?

Reply:
Ugh.. Yes. I'll update the commit message.

Reply:
Done.